articles/azure/azs-how-create-vm-from-snapshot-portal.md

```diff
@@ -0,0 +1,154 @@
+---
+title: How to create a virtual machine from a disk snapshot using the UKCloud Azure Stack Hub portal
+description: Provides help for creating a virtual machine from a managed disk snapshot using the portal on UKCloud for Microsoft Azure
+services: azure-stack
+author: William Turner
+reviewer: William Turner
+lastreviewed: 05/05/2020
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: Create a Virtual Machine from a disk snapshot
+toc_sub3:
+toc_sub4:
+toc_title: Create a virtual machine from a disk snapshot - Portal
+toc_fullpath: Users/How To/Create a Virtual Machine from disk snapshot/azs-how-create-vm-from-snapshot-portal.md
+toc_mdlink: azs-how-create-vm-from-snapshot-portal.md
+---
+
+# How to create a virtual machine from a disk snapshot using the UKCloud Azure Stack Hub portal
+
+## Overview
+
+This article shows you how to use the UKCloud Azure Stack Hub portal to:
+
+* Take a snapshot of a managed disk
+
+* Create a new managed disk from the snapshot
+
+* Create a virtual machine from the new managed disk
+
+### Intended audience
+
+To complete the steps in this article, you must have appropriate access to a subscription in the Azure Stack Hub portal.
+
+## Creating a snapshot from a disk
+
+1. Log in to the Azure Stack Hub portal.
+
+    For more detailed instructions, see the [*Getting Started Guide for UKCloud for Microsoft Azure*](azs-gs.md).
+
+2. Click **All services** in the favourites panel, then select **Disks** under the *Compute* section.
+
+    ![All services - disks](images/azs-browser-allservices-disks.png)
+
+3. From the list, select the disk that you would like to create a snapshot from.
+
+    ![Disks blade](images/azs-browser-disks-list.png)
+
+    > [!NOTE]
+    > Snapshots can only be created from managed disks.
+
+4. In the new blade for the selected disk, click **Create snapshot**.
+
+    ![Create snapshot from disk - New Blade](images/azs-create-snapshot-disk.png)
+
+5. In the **Create snapshot** blade, enter the following information:
+
+    - **Name** - The name of the snapshot
+
+    - **Subscription** - This will be your UKCloud for Microsoft Azure subscription
+
+    - **Resource group** - Select an existing resource group, or create a new one by typing a name for your new resource group
```

Comment:
Select an existing resource group, or create a new one by clicking the **Create new** link and then typing a name for your new resource group in the pop out window.

Reply:
Thanks for highlighting this.